bindings/python/pinocchio/visualize/meshcat_visualizer.py

```diff
@@ -275,6 +275,17 @@ def display(self, q = None):
             # Update viewer configuration.
             self.viewer[visual_name].set_transform(T)
 
+    def captureImage(self):
+        """Capture an image from the Meshcat viewer and return an RGB array."""
+        try:
+            img = self.viewer.get_image()
+            img_arr = np.asarray(img)
+            return img_arr
+        except AttributeError:
+            warnings.warn("meshcat.Visualizer does not have the get_image() method."
```

Comment:
Could you directly provide the minimal version of Meshcat here (next version)?

Reply:
I'll ask rdeits if he can release on PyPI after merging and I'll report the version number

Reply:
Ok, he updated the version to `0.2.0`: https://github.com/rdeits/meshcat-python/commit/46fe99c3044f788c84d45d2c70c0483be38f80b6